index.html

```diff
@@ -1,311 +1,321 @@
+                        existing story over the course of a day without spamming their friends.</p>
+
+                </div>
+                <div class="languages">
+                    <ul class="flex-lang">
+                        <li><span>html</span></li>
+                        <li><span>css</span></li>
+                        <li><span>javaScript</span></li>
+                    </ul>
+                </div>
+                <div class="btn-see-projects">
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+        </section>
+
+        <section class="works card-4">
+            <div class="card-container-1">
+                <img class="img-version-mobile" src="/img/Snapshoot-Portfolio-4.png" alt="Snapshot Portfolio Image"
+                    >
+                <img class="img-version-desktop" src="/img/img-desktop-4.png" alt="Snapshot Portfolio Image">
+
+            </div>
+
+            <div class="grid-container-desktop order">
+                <div class="title">
+                    <h1 class="mobile-title">Multi-Post<br>Stories</h1>
+                    <h1 class="desktop-title">Uber Navigation</h1>
+                </div>
+                <div class="title experience">
+
+                    <ul class="flex-container-list">
+                        <li>
+                            <h2 class="mobile-version-h2">CANOPY</h2>
+                            <h2 class="desktop-version-h2">Uber</h2>
+                        </li>
+                        <li class="mobile-version-li">Back End Dev</li>
+                        <li class="desktop-version-li1">Lead Developer</li>
+                        <li class="mobile-version-li2">2015</li>
+                        <li class="desktop-version-li2">2018</li>
+                    </ul>
+                </div>
+                <div class="paragraph">
+                    <p class="mobile-para">A daily selection of privately<br> personalized reads; no accounts or
+                        sign-ups required.</p>
+                    <p class="desktop-para">A smart assistant to make driving more safe, efficient, and fun by unlocking
+                        your most expensive computer: your car.</p>
+
+                </div>
+                <div class="languages">
+                    <ul class="flex-lang">
+                        <li><span>html</span></li>
+                        <li><span class="desktop-version-span-container">Ruby on rails</span></li>
+                        <li><span>css</span></li>
+                        <li><span>javaScript</span></li>
+                    </ul>
+                </div>
+                <div class="btn-see-projects">
+                    <a href="#" class="button">See Project</a>
+                </div>
+            </div>
+
+        </section>
+
+        <section class="about">
```

Comment:
- [ ] I think it's better to fix the styling for the mobile version of the about me section. Your portfolio must be amazing for any size of the screen.
<img width="558" alt="Screen Shot 2022-08-27 at 11 13 35 PM" src="https://user-images.githubusercontent.com/47673311/187054592-54e256ba-5249-41e6-9bf8-3eec58cd620c.png">